saleor/plugins/webhook/tests/subscription_webhooks/test_create_deliveries_for_subscription.py

```diff
@@ -43,14 +44,22 @@ def test_category_updated(category, subscription_category_updated_webhook):
 def test_category_deleted(category, subscription_category_deleted_webhook):
     # given
     webhooks = [subscription_category_deleted_webhook]
+
+    category_query = Category.objects.filter(pk=category.id)
+    category_instances = [cat for cat in category_query]
+    category_query.delete()
+
     event_type = WebhookEventAsyncType.CATEGORY_DELETED
-    category_id = graphene.Node.to_global_id("Category", category.id)
+    category_id = graphene.Node.to_global_id("Category", category_instances[0].id)
 
     # when
-    deliveries = create_deliveries_for_subscriptions(event_type, category, webhooks)
+    deliveries = create_deliveries_for_subscriptions(
+        event_type, category_instances[0], webhooks
+    )
 
     # then
     expected_payload = json.dumps({"category": {"id": category_id}, "meta": None})
+    assert graphene.Node.from_global_id(category_id)[1] != "None"
```

Comment:
You could do that without converting `category_id` from global ID. You could just check if DB's id is not None. 
Something like 👇 
```
category_instances = [cat for cat in category_query]
db_category_id = category_instances[0]
...
# then
assert db_category_id
expected_payload
...
```